SMLHelper/Assets/FishPrefab.cs

```diff
@@ -69,7 +69,11 @@ public sealed override GameObject GetGameObject()
             forces.handleGravity = true;
             forces.underwaterDrag = 1f;
             forces.underwaterGravity = 0;
+#if BELOWZERO
+            forces.waterDepth = Ocean.GetOceanLevel();
+#elif SUBNAUTICA
             forces.waterDepth = Ocean.main.GetOceanLevel();
+#endif
```

Comment:
```cs
#if BELOWZERO
    forces.waterDepth = Ocean.GetOceanLevel();
#else 
    forces.waterDepth = Ocean.main.GetOceanLevel();
#endif
```

Reply:
If both `SUBNAUTICA` and `BELOWZERO` are false, default to `SUBNAUTICA` instead of throwing a compile error.

Reply:
There are no other build configurations in the project anymore.  
The default RELEASE and DEBUG build configurations were removed as part of this effort.  
There is no reason to ever encounter the scenario of both of these if-directives returning false.

The reason I choose not to use `#else` is so that it is easy to search the code for Subnautica and BelowZero specific code by searching for "if SUBNAUTICA" and "if BELOWZERO".  
We will lose that if we start using the `#else` directive.